recipes/ngs-bits/meta.yaml

```diff
@@ -0,0 +1,38 @@
+package:
+  name: ngs-bits
+  version: 2018_03
+
+build:
+  number: 0
+
+source:
+  git_rev: d6ade58f2fd4b663e216bb7809afb856eeebd773
+  git_url: https://github.com/imgag/ngs-bits.git
```

Comment:
Upstream does have a 2018_03 release. Any reason why you need to check out a git revision directly? (Either explain git necessity with comment or switch to release tar ball please)